app/views/templates/search/search_object.rb

```diff
@@ -151,8 +155,19 @@ def content
                 }
               ]
             }),
-
-            description: render_document_show_field_value(document, 'proxies.dcDescription'),
+            description: data_section( {
+               :title => 'site.object.meta-label.description',
+               :sections  => [
+                 {
+                   :title  => false,
+                   :collected => render_document_show_field_value(document, 'proxies.dcDescription')
+                 },
+                 {
+                   :title   => false,
+                   :collected  => render_document_show_field_value(document, 'proxies.dctermsTOC')
```

Comment:
Line is too long. [98/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.